ui/webui/test/check-language

```diff
@@ -114,5 +114,20 @@ class TestLanguage(anacondalib.VirtInstallMachineCase):
         l.check_selected_locale("de_DE")
         b.wait_attr("html", "dir", "ltr")
 
+    def testLanguageScreenHiding(self):
+        self.restore_file('/run/anaconda/anaconda.conf')
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+
+        m.execute("sed -i 's/type = BOOT_ISO/type = LIVE_OS/g' /run/anaconda/anaconda.conf")
+
+        # Need to wait for storage-devices step because installation-language was hidden
+        i.open(step="storage-devices")
+
+        # Expect Storage devices page
```

Comment:
Instead of checking the contents of the page it is cleaner to:
self.assertEqual(self.get_current_page(), i.steps.STORAGE_DEVICES)

Reply:
I send my latest code here, but i dot do this step in the code. Because if we use original i.open i cant be anything else than storage-devices or timeout.